templates/html/header.html

```diff
@@ -1,10 +1,11 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
```

Comment:
I don't think this is correct. When choosing in the setting another language (see `OUTPUT_LANGUAGE`)  shouldn't this change as well?

Reply:
indeed - this ought to be updated to correspond to the language of the output.   I'm not sure my skills are up to this, beyond some hacky inline JS if/then function.

Reply:
I don't think a JS solution is here a good solution as it probably will kick in after the page is loaded / during the load.

For XML and DocBook in #7825 the `lang` attribute has been added and when placing a placeholder `lang=$lang` in the default HTML header is is possible to replace this `$lang` attribute during the generation of the HTML pages by the correct language code (analogous to e.g. `$projectbrief`)

the question though is:
- Are all the language codes (for the doxygen supported languages) for XML the same as for HTML?
   The language codes are defined in the function `trISOLang()` in the different `translator_...h` files.

when this is the case the above solution will work, in the other case a new translation function has to be defined (or an optional boolean argument has to be added).